cmake/modules/AddCephTest.cmake

```diff
@@ -51,17 +53,41 @@ function(add_ceph_unittest unittest_name)
   target_link_libraries(${unittest_name} ${UNITTEST_LIBS})
 endfunction()
 
-function(add_tox_test name tox_path)
+function(add_tox_test name)
   set(test_name run-tox-${name})
+  set(venv_path ${CEPH_BUILD_VIRTUALENV}/${name}-virtualenv)
+  cmake_parse_arguments(TOXTEST "" "TOX_PATH" "TOX_ENVS" ${ARGN})
+  if(DEFINED TOXTEST_TOX_PATH)
+    set(tox_path ${TOXTEST_TOX_PATH})
```

Comment:
So that's how you deal with optional/default values in Cmake... I have little experience with Cmake syntax :S